pyproject.toml

```diff
@@ -16,6 +16,7 @@ dependencies = [
     "sqlalchemy>=1.3",
     "annotated-types",
     "typing-extensions",
+    "pyensmallen",
```

Comment:
`pyensmallen` is going to be an optional dependency, so we don't need to list it in pyproject.toml